ksql-engine/src/test/java/io/confluent/ksql/integration/JoinIntTest.java

```diff
@@ -33,37 +33,53 @@
 
 
   private String orderStreamTopic = "OrderTopic";
+  private String orderStreamName = "Orders_json";
   private OrderDataProvider orderDataProvider;
   private Map<String, RecordMetadata> orderRecordMetadataMap;
 
+  private String orderStreamTopicAvro = "OrderTopicAvro";
+  private String orderStreamNameAvro = "Orders_avro";
+  private Map<String, RecordMetadata> orderRecordMetadataMapAvro;
+
   private String itemTableTopic = "ItemTopic";
+  private String itemTableName = "Item_json";
   private ItemDataProvider itemDataProvider;
   private Map<String, RecordMetadata> itemRecordMetadataMap;
+
+  private String itemTableTopicAvro = "ItemTopicAvro";
+  private String itemTableNameAvro = "Item_avro";
+  private Map<String, RecordMetadata> itemRecordMetadataMapAvro;
+
   private final long now = System.currentTimeMillis();
 
   @Before
   public void before() throws Exception {
-    testHarness = new IntegrationTestHarness(DataSource.DataSourceSerDe.JSON.name());
+    testHarness = new IntegrationTestHarness();
     testHarness.start();
     Map<String, Object> ksqlStreamConfigProps = new HashMap<>();
     ksqlStreamConfigProps.putAll(testHarness.ksqlConfig.getKsqlStreamConfigProps());
     // turn caching off to improve join consistency
     ksqlStreamConfigProps.put(StreamsConfig.CACHE_MAX_BYTES_BUFFERING_CONFIG, 0);
     ksqlStreamConfigProps.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "latest");
-    ksqlContext = KsqlContext.create(new KsqlConfig(ksqlStreamConfigProps));
+//    ksqlContext = KsqlContext.create(new KsqlConfig(ksqlStreamConfigProps));
+    ksqlContext = KsqlContext.create(new KsqlConfig(ksqlStreamConfigProps), testHarness.schemaRegistryClient);
 
     /**
      * Setup test data
      */
     testHarness.createTopic(itemTableTopic);
+    testHarness.createTopic(itemTableTopicAvro);
     itemDataProvider = new ItemDataProvider();
 
     itemRecordMetadataMap = testHarness.publishTestData(itemTableTopic, itemDataProvider, now -500);
+    itemRecordMetadataMapAvro = testHarness.publishTestData(itemTableTopicAvro, itemDataProvider, now -500, DataSource.DataSourceSerDe.AVRO.name());
```

Comment:
I think this might fail checkstlye now that the line length check has been turned on. Maybe other lines, too.